### PR TITLE
New Port: mReader

### PR DESCRIPTION
**Pull request by booYah187**
Game Information
Title: mReader
URL: https://github.com/booYah187/mReader

Instructions: This port is ready to run.

Controls
| Button | Action |
||| 
|DPAD/LStick/RStick| Scroll/Page|
|A/L1/R2| Enter|
|B| Back|
|X| Settings|
|Y/Start| TOC|

CFW Tests
[X] AmberELEC
[X] ArkOS
[X] JELOS / ROCKNIX
[X] muOS 

Architecture
[X] aarch64
[X] Panfrost (Mainline) 

Resolution Tests
Test all major resolutions:
[] 480x320 (Optional)
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/mreader/gameinfo.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./mReader.sh</path>
+    <name>mReader</name>
+    <desc> .epub and .txt reader for PortMaster capable devices. Ported from pixel-reader by ealang. Add your books to /ports/mreader/Books, or just navigate via gui to your preferred location. Additional fonts can be added to /mreader/resources/fonts.
+    </desc>
+    <releasedate>19960101T000000</releasedate>
+    <developer>https://github.com/booYah187/mReader</developer>
```
Comment (Cebion) [inline ports/mreader/gameinfo.xml:9]: Don't use  URL as developer name. Persons or Groups or unknown.

File: ports/mreader/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <publisher></publisher>
+    <genre>Tool</genre>
+    <image>./mReader/cover.png</image>
```
Comment (Cebion) [inline ports/mreader/gameinfo.xml:12]: foldername needs to be lowercase

File: ports/mreader/mreader/mReader.sh
Comment (Cebion) [inline ports/mreader/mreader/mReader.sh:1]: You have two mReader.sh files.

File: ports/mreader/port.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": 3,
+    "name": "mReader.zip",
```
Comment (Cebion) [inline ports/mreader/port.json:3]: zipname and folder name need to be lowercase

File: ports/mreader/port.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "items": [
+        "mReader.sh",
+        "mReader"
```
Comment (Cebion) [inline ports/mreader/port.json:6]: lowercase

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/mreader/port.json (hunk 4 above)
Comment (booYah187) [reply ports/mreader/port.json]: Corrections committed.

**Review by Cebion — APPROVED** [review]

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**